### ISSUE TITLE
TSTool command file to convert to csv is not working correctly

### ISSUE THREAD

**Issue by smalers** [OpenWaterFoundation/owf-data-co-counties#4]
Despite formatting columns as text TSTool is treating FIPS_ID as number values > 100 have decimal, even when treating as integer.

**Comment by smalers** [comment]
This is not a TSTool issue.  TSTool run at debug level 1 shows the FIPS_ID for Pueblo as being a number internal to Excel.  The issue is if you want a number to really be treated as text, the format of the cell must first be set to text before entering the number.  See:  [A number formatted as text is really a number](https://answers.microsoft.com/en-us/office/forum/office_2010-excel/a-number-formatted-as-text-is-really-a-number/085bb189-342b-4eed-bc33-2f149c6db244).  I'm kicking this back to Kristin to resolve.

**Comment by smalers** [comment]
My command file is attached but Kristin should add comments as appropriate and write the output to the "data" folder.   I had to add .txt to the end so that GitHub would accept this attachment.  This command file outputs a log file so *.log should be added to .gitignore.
[export-from-excel.TSTool.txt](https://github.com/OpenWaterFoundation/owf-data-co-counties/files/1484150/export-from-excel.TSTool.txt)

**Comment by smalers** [comment]
TSTool 12.05.00 now contains a way to check string properties for substrings using "contains".  This can be used to confirm that the original Excel formatting was correct for text fields containing numbers.  See the attached command file.
[export-from-excel.TSTool.txt](https://github.com/OpenWaterFoundation/owf-data-co-counties/files/1486596/export-from-excel.TSTool.txt)

**Comment by kswaim** [comment]
After reading "A number formatted as text is really a number", it appears that it would be necessary to re-enter by hand the values into a new column that is pre-formatted as text.  This has the potential to increase errors in entering by hand.  I realized that when I saved the original data downloads as csv, I just opened them up in Excel to start creating the dataset.  Instead, it is best to open a blank Excel document and then select Data/Get External Data/From Text.  In the wizard, you have the option to format the columns before they are imported.  I formatted both the FIPS_ID and the DOLA_LG_ID as text and then imported the data.  One obvious way to tell if the data were imported as text is to note if they are left-justified in the column.  I then saved the dataset and opened it in TSTool, where it was confirmed that the FIPS_ID and DOLA_LG_ID are text.
